app/virtualizerRetargeting/virtualizerConfig.ini

```diff
@@ -8,5 +8,5 @@ deadzone        0.35
 playerOrientationPort_name    /playerOrientation:o
 robotOrientationPort_name     /robotOrientation:i
 rpcWalkingPort_name           /walkingRpc
-scale_X                       2.0
-scale_Y                       1.0
+scale_X                       3.0
+scale_Y                       2.0
```

Comment:
Why here, we changed the `scale_Y`? As you can see here in the following line of the code:
https://github.com/robotology/walking-teleoperation/blob/9bc40b0f6e301362a5ac4b7fd0c14bb52b21b744/modules/Virtualizer_module/src/VirtualizerModule.cpp#L225
By setting this value to 2, actually if the user rotate 90 degrees inside the virtualizer, the robot will rotate about 180 degrees. At least, this value will be provided to the unicycle planner.

Reply:
This also explains why when we were teleoperating the robot with @GiulioRomualdi, the robot was rotating a lot and it was hard to control the rotational motion of the robot teleoperated.

Reply:
Talking with @S-Dafarra , it came out that the increasing the  scaling factor does not mean two times the amount of rotation, and we both agree that the system will more reactive to the user.

As we tested several times before on iCub 2.5, the value `1`, here also I suggest keeping the value `1` as the default value.

Reply:
> As we tested several times before on iCub 2.5, the value `1`, here also I suggest keeping the value `1` as the default value.

It is a number that after some trials seemed to be working better. It is probably a little low as well, considering that we are walking at a speed slower than what we achieve with the joystick.

Reply:
Following the f2f discussion, since we did the experiment with the value in this config file, and we keep it for now here, Later, a tuning session can be done to check this parameter.